numpy/testing/noseclasses.py

```diff
@@ -298,19 +300,14 @@ def configure(self, options, config):
                                   if p.name != self.to_unplug]
 
 
-class KnownFailureTest(Exception):
-    '''Raise this exception to mark a test as a known failing test.'''
-    pass
-
-
-class KnownFailure(ErrorClassPlugin):
+class KnownFailurePlugin(ErrorClassPlugin):
```

Comment:
This is probably safe, at least judging from Scipy, but it worries me a bit just the same. Too bad it wasn't private. I don't know what all other projects use `numpy.testing`.